.azure-pipelines.yml

```diff
@@ -467,3 +467,91 @@ jobs:
     env:
       PYPI_CREDENTIALS: $(PYPI_CREDENTIALS)
     displayName: 'Upload to PyPI'
+
+
+#################################################################################
+# Step 5 : macOS build and tests
+
+
+- job: 'macos_test'
+  displayName: 'MacOS tests'
+  dependsOn: s0_build_wheel
+  timeoutInMinutes: 60
+  pool:
+    vmImage: 'macOS-10.15'
+  variables:
+    PGINSTALLATION: '/usr/local/bin'
+  steps:
+  - checkout: self
+    submodules: true
+  - task: UsePythonVersion@0
+    inputs:
+      versionSpec: '$(python.version)'
+  - download: current
+    artifact: wheel
+  - bash: |
+      set -eux
+      cd packaging/macOS
```

Comment:
you can use `workingDirectory: $(Build.SourcesDirectory)/packaging/macOS` instead